backend/kale/common/podutils.py

```diff
@@ -145,6 +145,8 @@ def _get_mount_path(container, volume):
 def _list_volumes(client, namespace, pod_name, container_name):
     pod = client.read_namespaced_pod(pod_name, namespace)
     container = _get_pod_container(pod, container_name)
+    snapshotclass_provisioners = list_snapshotclass_storage_provisioners()
+    provisioner_names = get_snapshotclass_provisioners_names()
```

Comment:
Can you move these calls directly where they are used? Since both `snapshotclass_provisioners` and `provisioner_names` are used only once, I would avoid doing this assignment and just call the functions later on